### PR TITLE
update SNFR20 with direct link to GH team creation

### PR DESCRIPTION
**Pull request by matebarabas**
# Overview/Summary

## This PR fixes/adds/changes/removes

Update SNFR20 with direct link to GH team creation and additional clarification on steps of team creation and their assignment to parent teams.

Related issue: #479

### Breaking Changes

n/a

## As part of this Pull Request I have

- [x] Read the Contribution Guide and ensured this PR is compliant with the guide
- [x] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Verified-Modules/pulls)
- [x] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Verified-Modules/issues) or ADO Work Items (Internal Only)
- [x] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Verified-Modules/)
- [x] Ensured PR tests are passing
- [x] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

**Review by eriqua — DISMISSED** [review]
:ok:

File: docs/content/specs/shared/_index.md [hunk 1]
```diff
@@ -506,22 +506,34 @@ Unless explicitly requested and agreed, members of the AVM core team or any PG t
...
-| `<hyphenated module name>-module-owners-bicep`       | Module Owners of the <module name> AVM Bicep <resource/pattern> module       | **Write**   | Assignment to the **`avm-technical-reviewers-bicep`** parent team. | Need to work in a fork. |
-| `<hyphenated module name>-module-contributors-bicep` | Module Contributors of the <module name> AVM Bicep <resource/pattern> module | **Triage**  | **`avm-module-contributors-bicep`** parent team.                   | Need to work in a fork. |
+| GitHub Team Name                                     | Description                                    | Permissions | Permissions granted through                                        | Where to work?          |
+|------------------------------------------------------|------------------------------------------------|-------------|--------------------------------------------------------------------|-------------------------|
+| `<hyphenated module name>-module-owners-bicep`       | AVM Bicep Module Owners - \<module name>       | **Write**   | Assignment to the **`avm-technical-reviewers-bicep`** parent team. | Need to work in a fork. |
```
Comment (arnoldna) [inline docs/content/specs/shared/_index.md:517]: Remove the "\" before <module name>

**Review by arnoldna — APPROVED** [review]

**Comment by matebarabas** [comment]
@eriqua, could you please review & approve it again? I was still polishing it based on Nate's feedback. I just committed the last changes. Thanks!

File: docs/content/contributing/q-and-a.md [hunk 2]
```diff
@@ -79,7 +79,7 @@ You can propose a new module [here][ModuleProposal].
 
 ### How long will it take for someone to respond and a module to be created/updated and published?
 
-Whilst there are SLAs defined for providing [support][ModuleSupport] for existing modules, there are currently no SLAs in place for the creation of new modules. The AVM core team is a small team and is currently working on automating the module creation process to make it as easy as possible for module owners to create and publish modules on their own.
+While there are SLAs defined for providing [support][ModuleSupport] for existing modules, there are currently no SLAs in place for the creation of new modules. The AVM core team is a small team and is currently working on automating the module creation process to make it as easy as possible for module owners to create and publish modules on their own.
```
Comment (eriqua) [inline docs/content/contributing/q-and-a.md:82]: Why this change? I think `whilst` is intentional. Same for other occurrences

Comment (matebarabas) [reply docs/content/contributing/q-and-a.md:82]: As it turns out it's UK English, whereas we write our documentation in US English. 😊

Comment (eriqua) [reply docs/content/contributing/q-and-a.md:82]: Goodness, gracious, me

File: docs/content/specs/shared/_index.md [hunk 3]
```diff
@@ -506,22 +506,34 @@ Unless explicitly requested and agreed, members of the AVM core team or any PG t
...
+Direct link to create a new GitHub team and assign it to its parent: [Create new team](https://github.com/orgs/Azure/new-team)
+
+Fill in the values as follows:
+
+- **Team name**: Following the naming convention described above, use the value defined in the module indexes.
+- **Description**: Follow the guidance above (see the Description column in the table above).
+- **Parent team**: Follow the guidance above (see the Permissions granted through column in the table above).
+- **Team visibility**: `Visible`
+- **Team notifications**: `Enabled`
+{{< /hint >}}
```
Comment (eriqua) [inline docs/content/specs/shared/_index.md:535]: What about a screenshot here? If so with usual style, dark mode with light green boxes to highlight content

Comment (matebarabas) [reply docs/content/specs/shared/_index.md:535]: I would avoid that. There's already a counter argument of some of the requirements being too lengthy. This one is definitely one of them. Adding a screenshot wouldn't add any value, as the UI is very simple, it only consists of these points listed. Also, adding a screenshot would need to be in addition to the text explaining (and not instead of), as the guidance is different for Bicep vs. TF. Hence, I would really not like to put one here. If the feedback is that this still needs further explanation, it should be part of a future module owners' guide.

Comment (eriqua) [reply docs/content/specs/shared/_index.md:535]: Fair enough, I'll approve since this is actually specs and I agree it's already too long as it is. Some of this content should probably be moved to the contribution guidelines for module owners instead. Let's rediscuss screenshots at that time

**Review by eriqua — APPROVED** [review]
:ok:
